chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -26,3 +26,8 @@ service pacemaker stop
 echo "No HA setup found..."
 
 <% end %>
+
+for bridge in $(ovs-vsctl --real list-br); do
+    echo "Resetting fail-mode on ovs-bridge $bridge"
+    ovs-vsctl del-fail-mode $bridge
```

Comment:
maybe we want to explicitly set the fail mode to *standalone* (`ovs-vsctl set-fail-mode bridge standalone`). This way openvswitchd can get over the missing controller (neutron agent).